src/index.html

```diff
@@ -5,9 +5,39 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <body class="body">
+    <div class="card" data-qa="card">
+      <img src="./images/imac.jpeg" alt="Apple A1419"
+      class="card__image">
+      <h2 class="card__name">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h2>
+      <p class="card__product-code">Product code: 195434</p>
+      <div class="card__star-rating">
+        <div class="stars stars--5">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+
+        <p class="card__reviews">
+          Reviews: 5
+        </p>
+      </div>
+
+      <div class="card__price">
+        <p class="card__price-text">Price:</p>
+        <p class="card__price-cost">$2,199</p>
+      </div>
+      <a href="#" class="card__button-buy"
+      data-qa="hover">Buy</a>
```

Comment:
Change the 'card__button-buy' class to 'card__button' for simplicity and consistency with other class names